src/dox/Dox.hx

```diff
@@ -65,6 +65,9 @@ class Dox {
 			@doc("Disable markdown rendering")
 			["--no-markdown"] => function() cfg.useMarkdown = false,
 
+			@doc("Show private fields")
```

Comment:
Is this accurate? Seems like it applies to types too, not just fields.

Reply:
That's true. Should I change the description or remove the inclusion of private types?

Reply:
I'd say update the description, if you're going to show private fields you might as well show private types too.

Reply:
All right, will push another commit in a moment